rcon/commands.py

```diff
@@ -492,7 +492,7 @@ def get_gamestate(self) -> List[str]:
         # Has no trailing "\n"
 
         result = self._get("gamestate", can_fail=False)
-        logger.info("Gamestate results:\n|%s|", result)
+        logger.debug("Gamestate results:\n|%s|", result)
```

Comment:
I think this can be removed completely. If I'm not mistaken, this should already be removed in master.